examples/web/assets/js/app.js

```diff
@@ -53,14 +53,21 @@ document.addEventListener("DOMContentLoaded", function (event) {
 	desktopJS.Container.addListener("layout-loaded", (e) => console.log("Layout loaded - static: " + e.layoutName));
 
 	subscribe();
+
+	// If url is provided a hash, try to navigate to bootstrap tab if exists
+	var activeTab = $('[href="' + location.hash + '"]');
+	activeTab && activeTab.tab('show');
```

Comment:
I'm not using this yet, but it will allow me to add from the message bus page a button to help the user open a child window with the message bus pill selected to make testing multi-window ipc easier.